spec/contract/snyk_spec.sh

```diff
@@ -51,7 +51,11 @@ Describe 'Contract test between the SDK and the Snyk CLI'
                 ./snyk-iac-rules template ./tmp --rule Contract
 
                 # replace the fixture path so it's correct
-                sed -i '' -e 's#/rules/Contract/fixtures#/tmp/rules/Contract/fixtures#' ./tmp/rules/Contract/main_test.rego
+                if [[ "$OSTYPE" == "darwin"* ]]; then
```

Comment:
I'm not sure how this worked before ... https://stackoverflow.com/questions/8683895/how-do-i-determine-the-current-operating-system-with-node-js